usr/share/rear/layout/save/Linux-arm/455_arm_bootloader.sh

```diff
@@ -0,0 +1,7 @@
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+[ "$BOOTLOADER" = "ARM" ] || return 0
+
+# Currently we just warn
+LogPrint "Warning: BOOTLOADER = ARM is just an Dummy!"
```

Comment:
This message is not actionable for the user, I suggest to replace / enhance it with a clear call to action like "Bootloader not managed by ReaR, please do XYZ yourself".